examples/browser/package.json

```diff
@@ -13,18 +13,26 @@
     "theia": "file:../../../Theia"
   },
   "scripts": {
-    "clean": "./node_modules/.bin/rimraf lib",
-    "lint": "./node_modules/.bin/tslint -c ../../config/tslint/tslint.json --project ./tsconfig.json",
+    "clean": "rimraf lib",
```

Comment:
if you have some package in your dependency, or devDependency and you installed it, you can always be sure that npm in npm-scripts will try to take local version from `./node_modules` for packages and `./node_modules/.bin/` for binaries